tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py26, py27, py33, py34, py35, pypy
+
+[testenv]
+passenv = HYPOTHESIS_STORAGE_DIRECTORY
+
+deps =
+  nose-cov
+  hypothesis
+
+commands =
+  nosetests -v --logging-level=DEBUG --with-cov --cov chardet --cov-config .coveragerc
```

Comment:
Coverage should not be calculated for `pypy` because it makes the built go from taking 3 minutes to 19 minutes.  That's why we had it setup with the environment variable check before.